src/sfconfig.h

```diff
@@ -113,9 +113,6 @@
 #define HAVE_X86INTRIN_H 0
 #endif
 
-#define CPU_IS_X86		(defined __i486__ || defined __i586__ || defined __i686__ || defined __x86_64__)
-#define CPU_IS_X86_64	(defined __x86_64__)
-
```

Comment:
I'd prefer:
```
#if (defined (__i486__) || defined (__i586__) || defined (__i686__) || defined (__x86_64__))
#define CPU_IS_X86		  1
#define CPU_IS_X86_64	  0
#elif defined __x86_64__
#define CPU_IS_X86		  0
#define CPU_IS_X86_64	  1
#else
#define CPU_IS_X86		  0
#define CPU_IS_X86_64	  0
#endif
```
and the rest of the code to stay the same. This should probably be in its own PR.